src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -22,14 +26,23 @@ public class SuggestTests : IntellisenseTestBase
         /// <returns>
         /// List of string representing suggestions
         /// </returns>
-        private string[] SuggestStrings(string expression, string contextTypeString = null)
+        private string[] SuggestStrings(string expression, PowerFxConfig powerFxConfig, string contextTypeString = null)
         {
             Assert.NotNull(expression);
 
-            var intellisense = Suggest(expression, contextTypeString);
+            var intellisense = Suggest(expression, powerFxConfig, contextTypeString);
             return intellisense.Suggestions.Select(suggestion => suggestion.DisplayText.Text).ToArray();
         }
 
+        private class EmptyEnumStore : EnumStore
+        {
+            private ImmutableDictionary<string, string> _enumDict = ImmutableDictionary<string, string>.Empty;
+            protected override ImmutableDictionary<string, string> EnumDict => _enumDict;
+        }
```

Comment:
We should discuss this pattern more before making it public. 
If you keep EnumStore as internal, I'm good with the change.